priam/src/main/java/com/netflix/priam/IConfiguration.java

```diff
@@ -73,6 +73,14 @@
      */
     public String getCassStopScript();
 
+    /**
+     * @return int representing how many seconds Priam should fail healthchecks for before gracefully draining (nodetool drain)
+     * cassandra prior to stop. If this number is negative then no draining occurs and Priam immediately stops Cassanddra
```

Comment:
Based on the description of this variable, when it is -ve number, should we still drain and stop C* but do not fail any health check before draining?

Reply:
I don't think so unless we want more configuration options. I think that a single second of wait time would not be a big deal for users who just want drain functionality. (So they'd set it to 1)

Reply:
Oh I see, so 0+ would do the right thing and "-1" is disabled, sure.